### PR TITLE
refactor: annotates 06

### PR DESCRIPTION
**Pull request by paschalidi**
based on https://github.com/dhis2/capture-app/pull/624

Hey 👋 This is again a small PR that annotates and sets some default variables

Since you for sure more experienced than me can you please have a look and let me know your thoughts?

### CODE REVIEW

File: src/core_modules/capture-core-utils/RulesEngine/VariableService/VariableService.js [hunk 1]
```diff
@@ -8,24 +8,24 @@ import getDateUtils from '../dateUtils/dateUtils';
...
     TrackedEntityAttribute,
     TrackedEntityAttributes,
-    Entity,
     Enrollment,
     Constants,
     OrgUnit,
     DateUtils,
+    RuleVariable,
 } from '../rulesEngine.types';
+import type { Entity } from '../../../capture-core/components/Relationships/relationships.types';
```
Comment (JoakimSM) [inline src/core_modules/capture-core-utils/RulesEngine/VariableService/VariableService.js:25]: Looked into this one. the type Entity should be replaced by TEIValues from rulesEngine.types.

File: src/core_modules/capture-core-utils/RulesEngine/rulesEngine.types.js [hunk 2]
```diff
@@ -109,35 +111,21 @@ export type ProgramRulesContainer = {
     constants?: ?Constants,
 };
 
-export type EventMain = {
```
Comment (JoakimSM) [inline src/core_modules/capture-core-utils/RulesEngine/rulesEngine.types.js:112]: Pretty sure you should keep this. This is eventdata not based on data elements.

File: src/core_modules/capture-core-utils/RulesEngine/rulesEngine.types.js [hunk 3]
```diff
@@ -109,35 +111,21 @@ export type ProgramRulesContainer = {
...
-    dueDate: string,
-};
-
 export type EventValues = {
     [elementId: string]: any,
 };
 
 export type InputEvent = EventValues | {};
 
-export type EventData = EventMain & EventValues;
```
Comment (JoakimSM) [inline src/core_modules/capture-core-utils/RulesEngine/rulesEngine.types.js:132]: same as above.

**Comment by paschalidi** [comment]
@JoakimSM applies the changes you suggested. Could you please have another look?

**Pull request merged**